app/jobs/storage_backup.rb

```diff
@@ -7,7 +7,7 @@ class StorageBackup < ApplicationJob
   def perform
     drive = Google::Apis::DriveV3::DriveService.new
     drive.authorization = Google::Auth::ServiceAccountCredentials.make_creds(
-      json_key_io: File.open('config/credentials/google-drive.json'),
```

Comment:
Waarom gebruiken we hier niet ook `Rails.application.credentials.google_drive_service_account.to_json`?

Reply:
Die had je niet verwacht, hè?

Reply:
Nee idd, maar fix zat al in: https://github.com/DispuutHamers/webapp/pull/643